forward_engineering/helpers/convertJsonSchemaToAvro.js

```diff
@@ -178,7 +178,12 @@ const filterSchemaAttributes = schema => {
 const convertMultiple = schema => {
 	const type = filterMultipleTypes(schema.type.map(type => {
 		if(_.isString(type)) {
-			return convertSchema({ ...schema, type });
+			const typeSchema = convertSchema({ ...schema, type });
+			if (_.isString(typeSchema)) {
+				return typeSchema;
+			}
+
+			return simplifySchema({ ..._.omit(typeSchema, GENERAL_ATTRIBUTES), type: typeSchema.type });
```

Comment:
if a result Avro schema of the type from multiple types is complex - we need to filter general attributes (they will be on the higher level) and simplify it (in case when there are no general attributes in it)